server/app/auth/ProfileFactory.java

```diff
@@ -99,6 +100,7 @@ private CiviFormProfileData create(Role[] roleList) {
     return p;
   }
 
+  @VisibleForTesting
```

Comment:
**suggestion (blocking):** Prefer `@RestrictedApi` since `@VisibleForTesting` is only informational.